### PR TITLE
Fixes of 2 and 6.

### PR DESCRIPTION
**Pull request by maximkozlov**
The following was fixed:
2. Default candle style values
6. 5% margin for min/max

### CODE REVIEW

**Comment by benmccann** [comment]
@maximkozlov thanks! could you rebase this change so that it can be merged?

File: src/scale.financialLinear.js [hunk 1]
```diff
@@ -155,8 +155,8 @@ module.exports = function(Chart) {
 				}
 			});
 
-			me.min = isFinite(me.min) ? me.min : DEFAULT_MIN;
-			me.max = isFinite(me.max) ? me.max : DEFAULT_MAX;
+            me.min = isFinite(me.min) ? me.min - me.min * 0.05 : DEFAULT_MIN;
```
Comment (benmccann) [inline src/scale.financialLinear.js]: use tabs to be consistent with spacing

**Comment by maximkozlov** [comment]
Conflict was resolved.

**Comment by benmccann** [comment]
Thanks. I've merged these changes in separate commits to fix the tabs vs spaces issue

**Pull request closed without merging**